core/contracts/oracle/implementation/Voting.sol

```diff
@@ -124,6 +125,8 @@ contract Voting is Testable, Ownable, OracleInterface, VotingInterface {
     // Max value of an unsigned integer.
     uint256 private constant UINT_MAX = ~uint256(0);
 
+    bytes32 public snapshotMessage = ECDSA.toEthSignedMessageHash(keccak256(abi.encodePacked("Sign For Snapshot")));
```

Comment:
Can you verify that these changes do not affect the ABI objects (including event hashes) for any existing contract methods and events BESIDES `snapshotCurrentRound()`?

I'd compile the before and after contracts, look at the ABI and in the `build/` directory and compare the JSON's using something like [this](http://jsondiff.com/)

Reply:
OK i looked into this. I diffed the compiled files, and there are a significant number of differences between files but i didnt see anything that would alert me to the hashes changing for events or other functions. I think we can be confident that those hashes wont change as long as the signatures in code did not change based on how the hashes are calculated: https://medium.com/mycrypto/understanding-event-logs-on-the-ethereum-blockchain-f4ae7ba50378

Reply:
Agree. We definitely missed this with the last round of changes, but I'm confident that the lookup hash for a function or event is _only_ affected when that function or event signature is changed. Since this PR only changes `snapshotCurrentRound()`, that should be the only one affected.

@nicholaspai thoughts?

Reply:
I think that's right but just want to be sure